packages/astro/src/core/app/node.ts

```diff
@@ -85,11 +85,11 @@ export class NodeApp extends App {
 		// @example "443,8080,80" => "443"
 		const forwardedPort = getFirstForwardedValue(req.headers['x-forwarded-port']);
 		const port =
-			forwardedPort ?? req.socket?.remotePort?.toString() ?? (isEncrypted ? '443' : '80');
+			forwardedPort ??  (isEncrypted ? '443' : '80');
 
 		const portInHostname = typeof hostname === 'string' && /:\d+$/.test(hostname);
-		const hostnamePort = portInHostname ? hostname : `${hostname}:${port}`;
-
+		const hostnamePort = portInHostname ? hostname : `${hostname}${port ? `:${port}` : ''}`;
```

Comment:
Isn't port always going to be truthy because the `port` variable above has a `??` fallback? Unless if it's intended that `forwardedPort` could be an empty string?